sdk/tests/deqp/functional/gles3/es3fLifetimeTests.js

```diff
@@ -100,8 +100,12 @@ es3fLifetimeTests.ScaleProgram.prototype.draw = function(vao, scale, tf, dst) {
 es3fLifetimeTests.ScaleProgram.prototype.setPos = function(buffer, vao) {
     gl.bindBuffer(gl.ARRAY_BUFFER, buffer);
     gl.bindVertexArray(vao);
-    if (buffer)
+    if (buffer) {
         gl.vertexAttribPointer(this.m_posLoc, NUM_COMPONENTS, gl.FLOAT, false, 0, 0);
```

Comment:
When array buffer is null, vertexAttribPointer doesn't work in WebGL, while it works in GLES. vertexAttribPointer in GLES has an extra  'pointer' argument. When bound array buffer is 0, vertexAttribPointer will attach null pointer to vertex attribute at index.